buildSrc/src/main/kotlin/io/spine/internal/BuildSettings.kt

```diff
@@ -0,0 +1,13 @@
+package io.spine.internal
+
+import org.gradle.jvm.toolchain.JavaLanguageVersion
+
+/**
+ * This object provides high-level constants, like version of JVM, to be used
+ * throughout the project.
```

Comment:
It cannot be used in the build script of `buildSrc` itself. Let's have another paragraph denoting this exception.